components/File/src/index.scss

```diff
@@ -0,0 +1,56 @@
+.denhaag-file {
+  display: flex;
+  color: inherit;
+  text-decoration: inherit;
+  border: var(--denhaag-file-border);
+
+  .utrecht-link {
```

Comment:
I think we don't want to override CSS properties like this, but we want to override the design tokens of the Utrecht Link if possible. (https://github.com/nl-design-system/utrecht/blob/main/components/link/css/_mixin.scss). @Yolijn What's your opinion about this?

Reply:
Yes I agree. We want to try not to override properties but design tokens.

Also we want to prevent Den Haag brand in the component. So no `--denhaag-space-xl` but `--denhaag-file-padding-inline-start` for example

In this case I would assume something like this

```css
.denhaag-file__link {
 --utrecht-link-padding-inline-start: var(--denhaag-file-link-padding-inline-start);
 background-position: ...
 background-image: ...
}

.denhaag-file__link:hover {
 --utrecht-link-color: var(--denhaag-file-hover-color, --utrecht-link-hover-color);
}
```